docs/source/user_guide/modeling_causal_relations/diagnosing_a_causal_model/index.rst

```diff
@@ -0,0 +1,15 @@
+Diagnosing a Causal Model
```

Comment:
Structure wise, this is probably the biggest mismatch between the 4-step recipe and GCMs, since the "refutation" step appears after the estimation.

Reply:
Yes, agree. No good answer. @emrekiciman @amit-sharma  Do you have an idea?

Reply:
This is fine, actually. Even in the 4-step recipe, we are now considering adding graph checks earlier in the pipeline (before identify_effect). The other refutation tests that depend on the final estimate, will remain at the end. 

I will keep this in mind as I go through my edits.